common/model-views.h

```diff
@@ -605,7 +605,7 @@ namespace rs2
             keep_calculating(true),
             depth_stream_active(false),
             resulting_queue_max_size(20),
-            resulting_queue(resulting_queue_max_size),
+            resulting_queue(static_cast<unsigned int>(resulting_queue_max_size)),
```

Comment:
Modifying the underlying type for resulting_queue_max_size can make this more readable

Reply:
I don't want to change any logic with this commit, this change is mostly to silence the compiler with sort of syntactic sugar. A comprehensive refactoring to really **fix** the casting should be done separately